app/routes/news._index/route.mdx

```diff
@@ -27,6 +27,33 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2025
 
+    <CollectionItem
+      className="maxw-none grid-container"
+      variantComponent={
+        <CollectionCalendarDate datetime='August 7, 2025' />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>Super-Kamiokande JSON Notices and Schema v4.5.0</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+
+          #### New Super-Kamiokande JSON Notices
+
+          The Super-Kamiokande Team and the GCN team are pleased to announce the availability of Super-Kamiokande notice types via the [new GCN](https://gcn.nasa.gov) in JSON format.  These notices can be streamed via [Kafka](https://gcn.nasa.gov/quickstart).  [SK_SN](https://gcn.gsfc.nasa.gov/sk_sn.html) notices remain available via GCN Classic and GCN Classic over Kafka.  The information included in the JSON-format Notices largely overlap with those in GCN Classic with some additional parameters: the number of detected inverse beta decay (IBD) events, the analysis pipeline used (snwatchorwit), and the type of processed sample (fullorpartial) used for event detection.
```

Comment:
There is missing spaces for the part in parenthesis:
`snwatch or wit`
`full or partial`